src/Database/ImgIndex.php

```diff
@@ -1350,12 +1350,13 @@ public function getDataSources($verbose=false, $enable=null) {
         }
 
         // Set defaults for verbose mode (JHelioviewer)
+        /* Some servers may not have AIA 171 data - better solution to be found
```

Comment:
> The issue I want raise is that server unconditionally sets AIA 171 dataset as default even in case it has no AIA 171 images.

Would you like me to comment these out on helioviewer.org?

Reply:
I think the best solution would be that this is configurable, with the default setting in the config file being AIA 171. Would be nice if it worked with no default dataset.

Reply:
wondering if this default is even needed

Reply:
This was a long time back now, but I assume it was failing because SDO wasn't there in the `$tree`
The front end doesn't send the request with `verbose=true` so this code looks safe to delete.

Reply:
Ah, verbose is for JHelioviewer.